src/dom/dom-manager.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * ## DOM Manager
+ *
+ * The Flow.js DOM Manager controls how different DOM elements -- including the attributes and attribute handlers provided by Flow.js for [variables](../../attributes-overview/), [operations](../../operations-overview/) and [conversion](../../converter-overview/), and those [you create](./attributes/attribute-manager/) -- are bound to the variable and operations [channels](../../channel-overview/) to link them with your project's model. See the [Epicenter architecture details](../../../creating_your_interface/arch_details/) for a visual description of how the DOM Manager relates to the [rest of the Epicenter stack](../../../creating_your_interface/).
```

Comment:
We should say "Two-way data binding from your interface to the channel" somewhere. Or maybe that belongs in the overall Flow.js objective (replacing "channel" with "model")